docs/src/main/sphinx/release.md

```diff
@@ -7,6 +7,7 @@
 :maxdepth: 1
 
 release/release-469
+```
```

Comment:
This should go in the previous commit

Reply:
Not sure how I messed up the commits so badly. Now its fixed.